Error trying to build whisper.cpp with openvino support.

```
cmake --build build -j --config Release
[  4%] Building CXX object CMakeFiles/whisper.openvino.dir/openvino/whisper-openvino-encoder.cpp.o                      /home/abdalnablse10/whisper.cpp/openvino/whisper-openvino-encoder.cpp: In function ‘int whisper_openvino_encode(whisper_openvino_context*, ggml_tensor*, ggml_tensor*)’:                                                                        /home/abdalnablse10/whisper.cpp/openvino/whisper-openvino-encoder.cpp:67:14: error: ‘struct ggml_tensor’ has no member named ‘n_dims’                                                                                                              67 |     if (mel->n_dims != 2) {
      |              ^~~~~~                                                                                             /home/abdalnablse10/whisper.cpp/openvino/whisper-openvino-encoder.cpp:69:28: error: ‘struct ggml_tensor’ has no member named ‘n_dims’                                                                                                              69 |             __func__, mel->n_dims);
      |                            ^~~~~~
/home/abdalnablse10/whisper.cpp/openvino/whisper-openvino-encoder.cpp:73:14: error: ‘struct ggml_tensor’ has no member named ‘n_dims’                                                                                                              73 |     if (out->n_dims != 2) {                                                                                           |              ^~~~~~
/home/abdalnablse10/whisper.cpp/openvino/whisper-openvino-encoder.cpp:75:28: error: ‘struct ggml_tensor’ has no member named ‘n_dims’                                                                                                              75 |             __func__, out->n_dims);                                                                                   |                            ^~~~~~                                                                               gmake[2]: *** [CMakeFiles/whisper.openvino.dir/build.make:76: CMakeFiles/whisper.openvino.dir/openvino/whisper-openvino-encoder.cpp.o] Error 1                                                                                                  gmake[1]: *** [CMakeFiles/Makefile2:350: CMakeFiles/whisper.openvino.dir/all] Error 2                                   gmake: *** [Makefile:146: all] Error 2
```

--- a/openvino/whisper-openvino-encoder.cpp
+++ b/openvino/whisper-openvino-encoder.cpp
@@ -64,15 +64,15 @@ int whisper_openvino_encode(
         return 0;
     }
 
-    if (mel->n_dims != 2) {
+    if (ggml_n_dims(mel) != 2) {
         fprintf(stderr, "%s: Error! mel ggml_tensor expected to have n_dims=2, but it has n_dims=%d\n",
-            __func__, mel->n_dims);
+            __func__, ggml_n_dims(mel));
         return 0;
     }
 
-    if (out->n_dims != 2) {
+    if (ggml_n_dims(out) != 2) {
         fprintf(stderr, "%s: Error! out ggml_tensor expected to have n_dims=2, but it has n_dims=%d\n",
-            __func__, out->n_dims);
+            __func__, ggml_n_dims(out));
         return 0;
     }
 

